test/install-fixtures.c

```diff
@@ -139,34 +140,30 @@ void fixture_helper_set_up_bundle(gchar *tmpdir,
 	testfilepath = g_build_filename(mountdir, "verify.txt", NULL);
 
 	/* Setup bundle content */
-	g_assert(test_prepare_dummy_file(tmpdir, "content/rootfs.ext4",
-			SLOT_SIZE, "/dev/zero") == 0);
-	g_assert(test_prepare_dummy_file(tmpdir, "content/appfs.ext4",
-			SLOT_SIZE, "/dev/zero") == 0);
-	g_assert(test_prepare_dummy_file(tmpdir, "content/bootloader.ext4",
-			SLOT_SIZE, "/dev/zero") == 0);
-	g_assert_true(test_make_filesystem(tmpdir, "content/rootfs.ext4"));
-	g_assert_true(test_make_filesystem(tmpdir, "content/appfs.ext4"));
-	g_assert_true(test_make_filesystem(tmpdir, "content/bootloader.ext4"));
-	if (manifest_content) {
-		g_autofree gchar *tmpfile = write_tmp_file(tmpdir, "content/manifest.raucm", manifest_content, NULL);
-		g_assert_nonnull(tmpfile);
-	} else {
-		g_assert(test_prepare_manifest_file(tmpdir, "content/manifest.raucm", options) == 0);
+	if (manifest_content || (options && options->slots)) {
```

Comment:
Why does this need to be guarded with `if (manifest_content)` ?

Reply:
It's not needed. `manifest_content` is optionally used instead of a generated manifest.